### PR TITLE
data itself is not an array and is a hash

### PR DESCRIPTION
**Pull request by trorabaugh**

### CODE REVIEW

**Comment by slavikm** [comment]
@trorabaugh let's discuss tomorrow - not sure I understand this script...

**Comment by trorabaugh** [comment]
not mine, existing script, was throwing out error because there was some code that popped off an array, data is a hash not array

**Comment by slavikm** [comment]
I would deprecate it - all of it exists directly in integration.

**Comment by meirwah** [comment]
so @slavikm @trorabaugh no need for this fix anymore?

**Comment by aviadl** [comment]
@slavikm @trorabaugh can you please handle this ?

File: Scripts/script-VirustotalIsMalicious.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 commonfields:
   id: dbbdc2e4-6105-4ee9-8e83-563a4b991a89
-  version: 12
```
Comment (Spivaka) [inline Scripts/script-VirustotalIsMalicious.yml]: should be -1

**Comment by Spivaka** [comment]
@slavikm @trorabaugh this is needed?
Either merge or close

**Comment by trorabaugh** [comment]
It was noticed for customer who was using this. I recommended the raw command, and just patched this one for the customer. I don't remember off the top of my head. If you want to close it thats fine but the command was breaking because it was not always an array.

**Comment by gshriki** [comment]
let's fix but also deprecate.

**Comment by Spivaka** [comment]
as mentu
ioned, unattended pr will be closed.

**Pull request closed without merging**